### DISCUSSION TITLE
Adding images/pdf docs to project

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#210] [Q&A] [answered]
Hey everyone, I've been problem solving for days on adding images and pdf documents to my project, but I keep getting errors, and the docs/images aren't showing up. I've tried it several different ways. Does anyone have any helpful tutorials or anything?

**Comment by jakegros** [comment]
Hey, what code have you been trying to use?  I find it easiest to add the images into an assets folder or images, then using that as your source.
I used this ``<img src="assets/Accessibility.svg.png">`` since my image is save in my "assets" folder and then '/' followed by the name of image. Let me know if this helps.

**Reply by Skye404** [reply]
I added the photo to an assets folder, but when I used the code you shared, the image still isn't showing up. Just the image icon is visible. This is the same issue I've been running into.

**Reply by jakegros** [reply, marked as answer]
I looked at your index file.  If this is the image you are trying to use, it would be:
``<img src="assets/css/School photo 2021.JPG">``

If it is another image, you would just use that image name instead.

**Reply by Skye404** [reply]
I tried it without the css part, and it didn't work. I tried it with the github URL link for the image next, but it didn't work. Ultimately, it worked with exactly how you have it. Thanks!

**Reply by jakegros** [reply]
That's fantastic. Yeah, it is quite picky. I'm figuring out most things as I go now that the context is a real project and not structured lessons.

**Comment by kassyoveson** [comment]
You could also try the code: <img src="my_image.png" alt="My image">

**Reply by Skye404** [reply]
@kassyoveson I can't see your code. Also, the image isn't showing up, even when I click on it and it tried to open in another tab.
